packages/components/src/font-size-picker/index.js

```diff
@@ -32,6 +32,8 @@ function FontSizePicker(
 		fontSizes = [],
 		disableCustomFontSizes = false,
 		onChange,
+		/** @type {'default' | '__unstable-large'} */
+		size = 'default',
```

Comment:
> Adding a size prop for this is mostly to orchestrate the roll-out. For this one I think it would make sense to eventually remove the prop and make it large-only.

If we plan on removing it, should we prefix this prop with `__unstable` or `__experimental`?

Is the roll-out strategy going to be similar to other style deprecations, or are just planning on "flipping a switch"?

Reply:
These are great questions. It prompted me to think about it again, and now I'm less sure about "eventually remove the prop and make it large-only". That might not be a good idea at all, it's hard to say at the moment. So I guess we should just treat this as a normal `size` prop for the time being.